src/components/App.tsx

```diff
@@ -2917,12 +2913,15 @@ class App extends React.Component<AppProps, AppState> {
           sceneY = midPoint.y;
         }
       }
-      this.startTextEditing({
-        sceneX,
-        sceneY,
-        insertAtParentCenter: !event.altKey,
-        container,
-      });
+      const hitElement = this.getElementAtPosition(sceneX, sceneY);
```

Comment:
I _think_ this works to prevent double-clicking from editing text on anything that isn't a text element.  If we don't want to text to be editable after the fact at all, let me know and I'll amend this (as well as the handler for hitting enter on an element)